pkg/controllers/repositoryserver/handler.go

```diff
@@ -284,7 +287,7 @@ func (h *RepoServerHandler) preparePodOverride(ctx context.Context) (map[string]
 	return podOverride, nil
 }
 
-func (h *RepoServerHandler) updateServerInfoInCRStatus(ctx context.Context, info crv1alpha1.ServerInfo) error {
+func (h *RepoServerHandler) updateServerInfoInCRStatus(ctx context.Context, podName string, serviceName string) error {
 	h.Logger.Info("Fetch latest version of RepositoryServer to update the ServerInfo in its status")
```

Comment:
This log is not part of this PR and we don't have to change it now. But it looks like a comment more than a log. And there are other instances of similar logs I see in these files.

Reply:
I think we should create a task just to correct the logs.